Backpack/src/main/res/drawable/chip_background.xml

```diff
@@ -1,21 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
-<selector xmlns:android="http://schemas.android.com/apk/res/android">
-  <item android:drawable="@drawable/chip_selected" android:state_pressed="true" android:state_selected="true"/>
-  <item android:drawable="@drawable/chip_default" android:state_selected="false"/>
-  <item android:state_selected="true">
-    <shape android:shape="rectangle">
-      <solid android:color="?chipSelectedBackgroundColor" />
-      <padding
-        android:bottom="@dimen/bpkSpacingMd"
-        android:left="@dimen/bpkSpacingBase"
-        android:right="@dimen/bpkSpacingBase"
-        android:top="@dimen/bpkSpacingMd" />
-      <corners
-        android:bottomLeftRadius="@dimen/bpkBorderRadiusPill"
-        android:bottomRightRadius="@dimen/bpkBorderRadiusPill"
-        android:topLeftRadius="@dimen/bpkBorderRadiusPill"
-        android:topRightRadius="@dimen/bpkBorderRadiusPill" />
-    </shape>
-  </item>
-</selector>
-
+<shape
```

Comment:
I would prefer to use the XML drawables instead of the functions. If you use the theme attributes there, that should work.